app/assets/javascripts/questionnaire_logic.js.erb

```diff
@@ -64,6 +64,19 @@ function toggle_shown_questions() {
   }
 }
 
+function toggle_shown_questions_for_dropdown () {
```

Comment:
why didn't you structure this the same way as the `toggle_hidden_questions_for_dropdown` function? Its the same right? except for the `!`

Reply:
Good point, I'll fix it.

Reply:
Actually, there's 3 differences.

- The `.find('option[data-hides-questions]')` instead of data-shows-questions
- The `.data('hides-questions')` instead of shows-questions
- and then the !this_is_checked instead of this_is_checked

I don't see a clean way to use one function for this so I'll keep them separate for now.

This code is only temporary anyways, until we have the react implementation of all of this.